tests/test_google_analytics_bookmarks.py

```diff
@@ -43,11 +43,10 @@ def calculated_states_by_stream(self, current_state):
                 expected_streams.remove(stream)
                 expected_streams.add(custom_streams_name_to_id[stream])
 
-        timedelta_by_stream = {stream: 15  # {stream_name: number_of_days, ...}
-                               for stream in expected_streams
-                               if stream != 'Acquisition Overview'}
         # try decreasing bookmark delta to compensate for data gaps in stream 'Acquisition Overview'
-        timedelta_by_stream['Acquisition Overview'] = 5
+        # {stream_name: number_of_days, ...}
+        timedelta_by_stream = {stream: 15 if stream != 'Acquisition Overview' else 5
+                               for stream in expected_streams}
```

Comment:
Is this because there is no data for this stream ?

Reply:
Yes, I have seen data gaps as large as 15 days for that stream.

Reply:
What if this gap changes over a period of time later for this stream or for some other stream ? Is it possible to get this number programmatically ?